app/config/services.yml

```diff
@@ -1,83 +1,47 @@
-    # Instead, the PHP class of the event subscriber includes a method that returns
-    # the list of events listened by that class.
-    # See https://symfony.com/doc/current/event_dispatcher.html#creating-an-event-subscriber
-    app.requirements_subscriber:
-        class: AppBundle\EventListener\CheckRequirementsSubscriber
-        arguments: ['@doctrine.orm.entity_manager']
-        tags:
-            - { name: kernel.event_subscriber }
-
-    # To inject the voter into the security layer, you must declare it as a service and tag it with security.voter.
-    # See https://symfony.com/doc/current/security/voters.html#configuring-the-voter
-    app.post_voter:
-        class: AppBundle\Security\PostVoter
-        public: false
-        tags:
-            - { name: security.voter }
-
-    # Uncomment the following lines to define a service for the Post Doctrine repository.
-    # It's not mandatory to create these services, but if you use repositories a lot,
-    # these services simplify your code:
-    #
-    # app.post_repository:
-    #     class:     Doctrine\ORM\EntityRepository
-    #     factory:   ['@doctrine.orm.entity_manager', getRepository]
-    #     arguments: [AppBundle\Entity\Post]
-    #
-    # // traditional code inside a controller
-    # $entityManager = $this->getDoctrine()->getManager();
-    # $posts = $entityManager->getRepository('AppBundle:Post')->findAll();
-    #
-    # // same code using repository services
-    # $posts = $this->get('app.post_repository')->findAll();
+    # makes classes in src/AppBundle available to be used as services
+    # this creates a service per class whose id is the fully-qualified class name
+    AppBundle\:
+        resource: '../../src/AppBundle/*'
+        # you can exclude directories or files
+        # but if a service is unused, it's removed anyway
+        exclude: '../../src/AppBundle/{Controller,Entity,Repository}'
+
+    # controllers are imported separately to make sure they're public
+    # and have a tag that allows actions to type-hint services
+    AppBundle\Controller\:
+        resource: '../../src/AppBundle/Controller'
+        public: true
+        tags: ['controller.service_arguments']
+
+    # Autowiring can't guess the constructor arguments that are not type-hinted with
+    # classes (e.g. container parameters) so you must define those arguments explicitly
+    AppBundle\Command\ListUsersCommand:
+        arguments:
+            $emailSender: '%app.notifications.email_sender%'
+
+    AppBundle\Twig\AppExtension:
+        arguments:
+            $locales: '%app_locales%'
+
+    AppBundle\EventListener\CommentNotificationSubscriber:
+        arguments:
+            $sender: '%app.notifications.email_sender%'
+
+    AppBundle\EventListener\RedirectToPreferredLocaleSubscriber:
+        arguments:
+            $locales: '%app_locales%'
+            $defaultLocale: '%locale%'
+
+    Twig_Extensions_Extension_Intl:
+        class:  Twig_Extensions_Extension_Intl
```

Comment:
If you upgrade to version 1.5.0 of Twig extensions, you can now use `Twig\Extensions\IntlExtension` (and so you won't need the `class` key